packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/utils.dart

```diff
@@ -106,14 +104,15 @@ class BorderLayout extends StatelessWidget {
 
 @immutable
 class Truncateable extends StatelessWidget {
-  const Truncateable({Key key, this.truncate, this.child}) : super(key: key);
+  const Truncateable({Key? key, this.truncate = false, this.child})
+      : super(key: key);
 
-  final Widget child;
+  final Widget? child;
```

Comment:
child should be non null. Look at all uses of the `Truncateable` widget. null is never passed in for `child`, so we know it should be non-nullable. Also, we are unconditionally null-asserting in the build method, which is another code smell that something should probably be non-nullable.

Reply:
done